ops/ansible/kube/cfg-botpress.yml

```diff
@@ -0,0 +1,146 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: botpress-config
```

Comment:
Why is this here? All bots already contain a botfile?

Reply:
This is just config declaration. If user doesn't want to use config then he should not mount it in dpl-botpress.yml. If he wants to use it but customized - he needs to correct here.